src/components/NetworkSelector/NetworkSelector.js

```diff
@@ -25,6 +25,10 @@ function NetworkSelector(props) {
         value: "optimism",
         text: "optimism",
       },
+      {
```

Comment:
I'd keep it out of the network selector for now until we're fully ready with it. Until then, we can manually type the url in.